### PR TITLE
Topic json upload rework

### PR DESCRIPTION
**Pull request by reiterl**
Rebased on #1912.
Needs datastore with action_worker route also for collection import_preview now on datastore [PR259 Open action_worker route for import_preview](https://github.com/OpenSlides/openslides-datastore-service/pull/259)

### CODE REVIEW

File: openslides_backend/action/actions/topic/import_.py [hunk 1]
```diff
@@ -1,51 +1,120 @@
...
+                    agenda_item = {
+                        field[7:]: value
+                        for field in self.agenda_item_fields
+                        if (value := entry.pop(field, None)) is not None
+                    }
+                    if agenda_item:
+                        agenda_item["id"] = self.topic_lookup.get_field_by_name(
+                            entry["title"], "agenda_item_id"
+                        )
+                        update_agenda_item_payload.append(agenda_item)
```
Comment (jsangmeister) [inline openslides_backend/action/actions/topic/import_.py:70]: Merge this with the code here:
https://github.com/OpenSlides/openslides-backend/blob/95d52ad8fd9782d81e6d4183dc4725086919e13c/openslides_backend/action/actions/agenda_item/agenda_creation.py#L82
Also, magic numbers (like the 7 here) are almost never a good idea

File: tests/system/action/topic/test_json_upload.py [hunk 2]
```diff
@@ -195,3 +158,77 @@ def test_json_uplad_permission(self) -> None:
             {"data": [{"title": "test"}], "meeting_id": 1},
             Permissions.AgendaItem.CAN_MANAGE,
         )
+
+
+class TopicJsonUploadForUseInImport(BaseActionTestCase):
```
Comment (jsangmeister) [inline tests/system/action/topic/test_json_upload.py:163]: Same comment here as in the account import: Move these to a separate file

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/topic/import_.py (hunk 1 above)
Comment (r-peschke) [reply openslides_backend/action/actions/topic/import_.py]: done

File: tests/system/action/topic/test_json_upload.py (hunk 2 above)
Comment (r-peschke) [reply tests/system/action/topic/test_json_upload.py:163]: see comment in account import

**Comment by r-peschke** [comment]
After discussion today it was decided to merge the PR like it is
@rrenkert @luisa-beerboom @m-schieder @jsangmeister (absent) @r-peschke

**Pull request merged**